utils/constants.ts

```diff
@@ -18,6 +18,28 @@ export const PFP_WL_CONTRACTS_MAINNET = [
   "0x76503062d78f4481be03c9145022d6a4a71ec0719aa07756f79a2384dc7ef16",
 ];
 
+export const NftCollections = [
+  {
+    imageUri:
+      "https://api.briq.construction/v1/preview/starknet-mainnet/0x42f5fc1f7b1845b307807fa05672177c9c5483cd4b7cb847000000000000000.png",
+    name: "Ducks everywhere",
+    externalLink: "https://briq.construction/product/ducks_everywhere",
+  },
+  {
+    imageUri: "https://assets.starkguardians.com/collection/1/1.png",
+    name: "StarkGuardians",
+    externalLink:
+      "https://unframed.co/collection/0x0727a63f78ee3f1bd18f78009067411ab369c31dece1ae22e16f567906409905",
```

Comment:
It's stark punks here, you use https://pyramid.market/collection/0x2d679a171589777bc996fb27767ff9a2e44c7e07967760dea3df31704ab398a